express/index.html

```diff
@@ -65,8 +68,7 @@ <h2>SysAdmin Like a Girl</h2>
 		<section id="about">
 			<div class="container" data-aos="fade-up">
 				<div class="row about-container">
-
-					<div class="col-lg-6 content order-lg-1 order-2">
+					<div>
```

Comment:
I think that code it's necessary here `class="col-lg-6 content order-lg-1 order-2"`. Because it's missing left and right margin to align text with next texts.

![image](https://user-images.githubusercontent.com/15144708/95004367-9b29ce00-05c0-11eb-9a8d-e45b441bb47d.png)

Reply:
I swear it's the last one change hahaha

Reply:
Done.